README.md

```diff
@@ -1,66 +1,76 @@
 # nfdiv-frontend
 
-## Getting Started.
+## Getting Started
 
 ### Prerequisites
 
 Running the application requires the following tools to be installed in your environment:
 
-  * [Node.js](https://nodejs.org/) v12.0.0 or later
-  * [yarn](https://yarnpkg.com/)
-  * [Docker](https://www.docker.com)
+* [Node.js](https://nodejs.org/) v12.0.0 or later
+* [yarn](https://yarnpkg.com/)
+* [Docker](https://www.docker.com)
 
-### Running the application.
+### Running the application
 
 Install dependencies by executing the following command:
 
  ```bash
-$ yarn install
+yarn install
  ```
+
 Bundle:
 
 ```bash
-$ yarn webpack
+yarn webpack
 ```
 
-Run:
+Start server:
 
 ```bash
-$ yarn start
+yarn start
 ```
 
-The applications's home page will be available at https://localhost:3001
+The application's home page will be available at [https://localhost:3001](https://localhost:3001).
 
 ### Running with Docker
 
 Create docker image:
 
 ```bash
-  docker-compose build
+docker-compose build
 ```
 
 Run the application by executing the following command:
 
 ```bash
-  docker-compose up
+docker-compose up
 ```
 
 This will start the frontend container exposing the application's port
 (set to `3001` in this template app).
 
-In order to test if the application is up, you can visit https://localhost:3001 in your browser.
+In order to test if the application is up, you can visit [https://localhost:3001](https://localhost:3001) in your browser.
 You should get a very basic home page (no styles, etc.).
 
 ## Developing
 
+**Note:** the application can be started without authentication by commenting out `PropertiesVolume` and `OidcMiddleware` in `src/main/server.ts`.
```

Comment:
I think we'll need to remove this as soon you won't be able to do anything without the log in. We need to keep the note about `yarn start:dev` though

Reply:
👍 Fixed in ecf5fc4